roboteam_ai/src/coach/CoachHeuristics.cpp

```diff
@@ -0,0 +1,91 @@
+//
+// Created by robzelluf on 3/21/19.
+//
+
+#include "CoachHeuristics.h"
+
+namespace rtt {
+namespace ai {
+namespace coach {
+
+double CoachHeuristics::maxDistanceFromBall = 6.0;
+
+double CoachHeuristics::calculateCloseToGoalScore(Vector2 position) {
+    double distanceFromGoal = (Field::get_their_goal_center() - position).length();
+
+    double score = exp(-0.1 * distanceFromGoal);
+    return score;
+}
+
+double CoachHeuristics::calculateShotAtGoalScore(Vector2 position, roboteam_msgs::World world) {
```

Comment:
There are quite some magic constants here. I think it could help if you define the scores as a constant somewhere (for example inside the function would be fine) to clearly show what the values mean.